src/pivotal-ui/components/forms/forms.scss

```diff
@@ -1549,6 +1551,8 @@ the way we see fit.
 ```
 */
 
+$form-input-error-height: 35px;
```

Comment:
Fair enough! I'd figured it was probably a function of the font-size and line height but wasn't sure.